libs/metrics/src/lib.rs

```diff
@@ -89,14 +89,14 @@ pub const DISK_WRITE_SECONDS_BUCKETS: &[f64] = &[
     0.000_050, 0.000_100, 0.000_500, 0.001, 0.003, 0.005, 0.01, 0.05, 0.1, 0.3, 0.5,
 ];
 
-pub fn set_build_info_metric(revision: &str) {
+pub fn set_build_info_metric(revision: &str, build_tag: &str) {
```

Comment:
Let's add `build_tag` to a test we have: https://github.com/neondatabase/neon/blob/e0ebdfc7ce82fce9f382031aab738c6d88d3fb23/test_runner/regress/test_build_info_metric.py#L5